src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,173 @@
+     * @param totalTask Total number of tasks in the tasks list.
+     */
+    public String addTask(Task newTask, int totalTask) {
+        return "Got it. I've added this task: \n" + newTask
+                + "\nNow you have " + totalTask + " tasks in the list.";
+    }
+
+    /**
+     * Prints a message to notify the user that a task is already completed.
+     *
+     * @param doneTask Task that is already completed
+     */
+    public String showAlreadyDone(Task doneTask) {
+        return "You have already completed the task: \n" + doneTask;
+    }
+
+    /**
+     * Prints a message to warn the user when the system is unable to recognise the command.
+     */
+    public String showInvalidCommand() {
+        return "OOPS!!! I'm sorry, but I don't know what that means :-(";
+    }
+
+    /**
+     * Prints a message to warn the user when the index input for a <code>delete</code> or <code>done</code>
+     * command is invalid.
+     */
+    public String showInvalidTaskIndex() {
+        return "OOPS!!! The task is unavailable.";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a deadline is invalid.
+     */
+    public String showInvalidDeadlineDesc() {
+        return "OOPS!!! Wrong description format for deadline.\n\n"
+                + "Please enter your deadline in the format of: \n"
+                + "  \"deadline (description) /by (date & time)\"\n\n"
+                + "The date and time should be in the format of: \n"
+                + "  \"[yyyy-MM-dd HHmm]\"";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a event is invalid.
+     */
+    public String showInvalidEventDesc() {
+        return "OOPS!!! Wrong description format for event.\n"
+                + "  \"event (description) /at (date & time)\"\n\n"
+                + "The date and time should be in the format of: \n"
+                + "  \"[yyyy-MM-dd HHmm]\"";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a todo is empty.
+     */
+    public String showEmptyTodoDesc() {
+        return "OOPS!!! The description of a todo cannot be empty.";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a deadline is empty.
+     */
+    public String showEmptyDeadlineDesc() {
+        return "OOPS!!! The description of a deadline cannot be empty.";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a event is empty.
+     */
+    public String showEmptyEventDesc() {
+        return "OOPS!!! The description of a event cannot be empty.";
+    }
+
+    /**
+     * Prints a message to warn the user when unable to read file.
+     */
+    public String showInvalidFile() {
+        return "OOPS!!! Unable to read the tasks in the file.";
+    }
+
+    /**
+     * Prints a message when the keyword the user is finding is unavailable.
+     */
+    public String showTaskNotFound() {
```

Comment:
Technically, you are not "showing" anything here, you are only retrieving the error message.
In light of this, can you think of a better name to reflect what the method/code is doing